### DISCUSSION TITLE
Disable update notification

### DISCUSSION THREAD

**Discussion by stetss** [texstudio-org/texstudio#2516] [General]
How to disable update notification in texstudio once and for all? We deploy texstudio to several computers but it's annoying switch off the update notification for every new user in %appdata%.

thx

**Comment by muzimuzhi** [comment]
Preferences -> General -> Uncheck "Automatically check every xx"
![image](https://user-images.githubusercontent.com/6376638/183456119-e96e69bd-6586-48e9-8c0d-73feb12aa56f.png)

**Reply by stetss** [reply]
Thank you, I'd like to switch the update notification off, but I can't do that for every single user manually.

**Reply by muzimuzhi** [reply]
That change is recorded in config file `%appdata$\texstudio.ini`. Is it feasible to update users' `texstudio.ini`?

**Reply by muzimuzhi** [reply]
Alternatively, it seems you can change the init setting for that checkbox (from `true` to `false`), rebuild texstudio (which will be done automatically by the github action), and distribute the modified installation package to users.
https://github.com/texstudio-org/texstudio/blob/92bcbe8fdd6db67684e746ccfd23d2b27511f0f8/src/configmanager.cpp#L507

**Comment by stetss** [comment]
To be honest, I thought there was a registry key or %ProgramData%\texstudio\settings.ini or something like that to suppress autoUpdateCheck notification once and for all, for each and every user on that computer, for current and updated versions of texstudio . . . . always and forever.
